code/__DEFINES/~skyrat_defines/antag_optin_defines.dm

```diff
@@ -0,0 +1,8 @@
+//defines for antag opt in objective checking
+//objectives check for all players with a value equal or greater than the 'threat' level of an objective then pick from that list
+//command + sec roles are always opted in regardless of opt in status
+
+#define YES_TEMP 1 //for temporary or otherwise 'inconvenient' objectives like kidnapping or theft
```

Comment:
autodoc these, put the comments above them like this: 

/// hhahahahaha im a comment